### PR TITLE
Extending installation doc part

### PR DESCRIPTION
**Pull request by ShavidzeT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,18 @@
 
 [![Version][npm-image]][npm-url] [![Travis CI][travis-image]][travis-url] [![Quality][codeclimate-maintainability-image]][codeclimate-maintainability-url] [![Coverage][codeclimate-c-image]][codeclimate-c-url] [![Dependencies][gemnasium-image]][gemnasium-url] [![Gitter][gitter-image]][gitter-url]
 
+Install with npm:
 
-## Installation
-
-```sh
+```bash
 npm i --save unused-files-webpack-plugin
```
Comment (tomchentw) [inline README.md]: Please help me change to `--save-dev`

File: README.md [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
 ```
 
+Install with yarn:
+
+```bash
+yarn add unused-files-webpack-plugin
```
Comment (tomchentw) [inline README.md]: Please add `--dev` between add and unused…

**Comment by ShavidzeT** [comment]
@tomchentw fix codeclimate analyzer.

**Comment by tomchentw** [comment]
Thank you! Will fix that

**Pull request merged**